doc/troubleshooting.md

```
@@ -0,0 +1,572 @@
+
+**エラー:**
+```
+FAILURE: Build failed with an exception.
+```
+
+**解決方法:**
+
+```bash
+# Gradle キャッシュをクリア
+cd android
+./gradlew clean
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)